### PR TITLE
US132260 - Implement put get storage postmessage

### PR DESCRIPTION
**Pull request by jcousins-cs**
https://rally1.rallydev.com/#/?detail=/userstory/610424404919&fdp=true

Feedback on code organization definitely welcome.

### CODE REVIEW

**Comment by sammeyerreed** [comment]
Should we add some tests? It seems like we're accumulating a decent amount of logic in these PRs

**Review by alogan519 — COMMENTED** [review]
What about splitting out a file for Storage? I think create the store there and have a put  and get method on there.
The main js file would handle registering/listening for events but use the Storage file for the get/put.

**Review by alogan519 — APPROVED** [review]
Nice job with all the new tests!

File: lti-postmessage-api.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+				subject: `${event.data.subject}.response`,
+				message_id: event.data.message_id,
+				...response
+			};
+		}
+		return response;
+	}
+
+	_logError(message) {
+		logger.error(null, message);
```
Comment (jcousins-cs) [inline lti-postmessage-api.js:37]: This is put into its own method so I can stub it out for testing

File: test/lti-launch.test.js [hunk 2]
```diff
@@ -2,6 +2,20 @@ import '../lti-launch.js';
 import { expect, fixture, html } from '@open-wc/testing';
 import { runConstructor } from '@brightspace-ui/core/tools/constructor-test-helper.js';
 
+window.D2L = {
+	LP: {
+		Web: {
+			UI: {
+				Flags: {
+					Flag() {
+						return true;
```
Comment (jcousins-cs) [inline test/lti-launch.test.js:11]: This is to mock the flag method used by the component

**Pull request merged**